File: tito-node-api/src/service/team/team.service.ts
import { ServiceBase } from '../_service.base';

// model imports
import { Team } from '../../model/tito/team';

// dependency imports
import { TeamRepository } from '../../repository/team/team.repository';
import { TeamValidator } from '../../validation/team.validator';

export class TeamService extends ServiceBase<Team> {
  // constructor
  constructor(protected _repository: TeamRepository,
    protected _validator: TeamValidator) {
      super(_repository, _validator);
  }
}

File: tito-java-api/src/com/tito/service/TeamService.java
package com.tito.service;

import com.tito.model.Team;
import com.tito.repository.TeamRepository;

public class TeamService extends TitoServiceBase<Team> {

	public TeamService(TeamRepository repository, JwtService jwtService) {
		super(repository, jwtService);
	}
}
